src/content/nist.gov/SP800-53/rev4/xml/NIST_SP-800-53_rev4_catalog.xml

```diff
@@ -55469,485 +55557,840 @@
-          <target>http://www.cnss.gov/Assets/pdf/nstissi_7003.pdf</target>
-          <title>NSTISSI No. 7003</title>
-       </citation>
-       <citation id="ref104">
-          <target>http://www.whitehouse.gov/omb/circulars_a130_a130trans4</target>
-          <title>OMB Circular A-130</title>
-       </citation>
-       <citation id="ref105">
-          <target>http://www.whitehouse.gov/omb/memoranda_m02-01</target>
-          <title>OMB Memorandum 02-01</title>
-       </citation>
-       <citation id="ref106">
-          <target>http://www.whitehouse.gov/sites/default/files/omb/assets/omb/memoranda/fy04/m04-04.pdf</target>
-          <title>OMB Memorandum 04-04</title>
-       </citation>
-       <citation id="ref107">
-          <target>http://www.whitehouse.gov/sites/default/files/omb/assets/omb/memoranda/fy2005/m05-24.pdf</target>
-          <title>OMB Memorandum 05-24</title>
-       </citation>
-       <citation id="ref108">
-          <target>http://www.whitehouse.gov/sites/default/files/omb/assets/omb/memoranda/fy2006/m06-16.pdf</target>
-          <title>OMB Memorandum 06-16</title>
-       </citation>
-       <citation id="ref109">
-          <target>http://www.whitehouse.gov/sites/default/files/omb/assets/omb/memoranda/fy2007/m07-11.pdf</target>
-          <title>OMB Memorandum 07-11</title>
-       </citation>
-       <citation id="ref110">
-          <target>http://www.whitehouse.gov/sites/default/files/omb/assets/omb/memoranda/fy2007/m07-18.pdf</target>
-          <title>OMB Memorandum 07-18</title>
-       </citation>
-       <citation id="ref111">
-          <target>http://www.whitehouse.gov/sites/default/files/omb/assets/omb/memoranda/fy2008/m08-22.pdf</target>
-          <title>OMB Memorandum 08-22</title>
-       </citation>
-       <citation id="ref112">
-          <target>http://www.whitehouse.gov/sites/default/files/omb/assets/omb/memoranda/fy2008/m08-23.pdf</target>
-          <title>OMB Memorandum 08-23</title>
-       </citation>
-       <citation id="ref113">
-          <title>OMB Memorandum 10-06-2011</title>
-       </citation>
-       <citation id="ref114">
-          <target>http://www.whitehouse.gov/sites/default/files/omb/memoranda/2011/m11-11.pdf</target>
-          <title>OMB Memorandum 11-11</title>
-       </citation>
-       <citation id="ref115">
-          <target>http://www.whitehouse.gov/sites/default/files/omb/memoranda/2011/m11-33.pdf</target>
-          <title>OMB Memorandum 11-33</title>
-       </citation>
-       <citation id="ref116">
-          <title>Personal Identity Verification (PIV) in Enterprise Physical Access Control System (E-PACS)</title>
-       </citation>
-       <citation id="ref117">
-          <target>http://www.us-cert.gov/ncas/alerts</target>
-          <title>US-CERT Technical Cyber Security Alerts</title>
-       </citation>
+      <resource id="ref001">
+         <title>5 C.F.R. 731.106</title>
+         <citation>
+            <text>5 C.F.R. 731.106</text>
```

Comment:
Should be `Code of Federal Regulations, Title 5, Administrative Personnel, Section 731.106, Designation of Public Trust Positions and Investigative Requirements (5 C.F.R. 731.106).`